doc/nrf/protocols/bt_mesh/fota.rst

```diff
@@ -7,8 +7,8 @@ Performing FOTA updates in Bluetooth mesh
    :local:
    :depth: 2
 
-In |NCS|, there is currently no support for Device Firmware Update (DFU) over Bluetooth® mesh when performing firmware over-the-air (FOTA) updates for your Bluetooth mesh devices and applications.
-However, support for FOTA updates with point-to-point DFU over Bluetooth Low Energy using the MCUmgr subsystem and the Simple Management Protocol (SMP) is available.
+The |NCS| supports FOTA updates with point-to-point DFU over Bluetooth® Low Energy using the MCUmgr subsystem and the Simple Management Protocol (SMP).
+It also supports :ref:`ble_mesh_dfu_samples` when performing firmware over-the-air (FOTA) updates for your Bluetooth mesh devices and applications.
```

Comment:
We need to mention that this feature is experimental yet because it comes with mesh 1.1 specification, which is not yet finalized.

Perhaps we should add such warning to the dfu samples docs?

Reply:
Would adding e.g. the following statement be sufficient/correct:

Bluetooth mesh DFU support in the |NCS| is experimental.